lib/broadside/target.rb

```diff
@@ -73,5 +75,15 @@ def env_vars
         end
       end.map { |k, v| { 'name' => k, 'value' => v } }
     end
+
+    def to_h
+      {
+        Target: @name,
```

Comment:
1. i think AWS uses `to_hash` not `to_h`; we should copy them given that this is an AWS related app.
2. capital letter keys? not super ruby ish; is there a reason?  
3. also slightly skeptical why it's `Target: @name` not just `Name: @name` - code is easier to read when variable names are in line with the keys (as far as they can be)